packages/graph/hash_graph/lib/graph/src/types/schema/combinator.rs

```diff
@@ -6,14 +6,14 @@ use crate::types::schema::{
 };
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(untagged)]
+#[serde(untagged, deny_unknown_fields)]
```

Comment:
I think we want `deny_unknown_fields` on *any* type which does not have `#[serde(flatten)]` at one of their fields. This avoids otherwise silently ignored fields when deserializing.